public/app/features/alerting/unified/components/rule-editor/RuleEditorSection.tsx

```diff
@@ -25,7 +25,7 @@ export const RuleEditorSection = ({
       </div>
       <div className={styles.content}>
         <FieldSet label={title} className={styles.fieldset}>
-          {description && <p className={styles.description}>{description}</p>}
+          {description && <div className={styles.description}>{description}</div>}
```

Comment:
why do we need this change?

Reply:
I had to change it in order to prevent the following error after the changes done to the _Need Help?_ tooltip. I didn't notice visible changes in other parts where it's being used but let me know if you have concerns about this change 
![image](https://github.com/grafana/grafana/assets/6271380/21b9fb47-c1e5-473b-b667-0a01bbe20602)